runtime/runtime.go

```diff
@@ -411,6 +411,14 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 		return fmt.Errorf("at least one address must be configured in runtime parameters")
 	}
 
+	if rt.Params.Addrs != nil {
+		for _, addr := range *rt.Params.Addrs {
+			if addr == ":8181" {
+				rt.logger.Warn("You are not binding to localhost. This could have security implications if not intended. Please see https://www.openpolicyagent.org/docs/latest/security/#interface-binding for more information.")
```

Comment:
"OPA running on a public (0.0.0.0) network interface. Unless you intend to expose OPA outside of the host, binding to the localhost interface (--addr localhost:8181) is recommended. See https://www.openpolicyagent.org/docs/latest/security/#interface-binding"

Reply:
This sounds better indeed will update :ok_hand: